unity-client/Assets/Scripts/MainScripts/DCL/Controllers/InputController/InputController.cs

```diff
@@ -4,6 +4,9 @@
 using UnityEngine;
 using UnityEngine.EventSystems;
 
+/// <summary>
+/// Mapping for Trigger actions
```

Comment:
Here we are missing an opportunity for explaining how trigger actions work. The same for other action types.

Reply:
I actually had the documentation for the actions there and then I decided (and later forgot about it) that it was better to put it in their classes. 

Gonna do it!